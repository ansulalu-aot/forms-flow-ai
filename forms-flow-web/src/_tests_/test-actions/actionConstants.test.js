import ACTION_CONSTANTS from '../../actions/actionConstants';

describe('ACTION_CONSTANTS', () => {
    test('should contain the correct properties', () => {
        expect(ACTION_CONSTANTS).toHaveProperty('SET_CURRENT_PAGE');
        expect(ACTION_CONSTANTS).toHaveProperty('SET_USER_AUTHENTICATION');
        expect(ACTION_CONSTANTS).toHaveProperty('SET_USER_TOKEN');
        expect(ACTION_CONSTANTS).toHaveProperty('SET_USER_ROLES');
        expect(ACTION_CONSTANTS).toHaveProperty('SET_USER_DETAILS');
        expect(ACTION_CONSTANTS).toHaveProperty('ERROR');
        expect(ACTION_CONSTANTS).toHaveProperty('IS_LOADING');
        expect(ACTION_CONSTANTS).toHaveProperty('SET_LANGUAGE');
        expect(ACTION_CONSTANTS).toHaveProperty('SET_SELECT_LANGUAGES');
        expect(ACTION_CONSTANTS).toHaveProperty('ROLE_IDS');
        expect(ACTION_CONSTANTS).toHaveProperty('ACCESS_ADDING');
    });

    test('should contain the correct properties of form', () => {
        expect(ACTION_CONSTANTS).toHaveProperty('FORM_DELETE');
        expect(ACTION_CONSTANTS).toHaveProperty('RESTORE_FORM_ID');
        expect(ACTION_CONSTANTS).toHaveProperty('RESTORE_FORM_DATA');
        expect(ACTION_CONSTANTS).toHaveProperty('FORM_HISTORY');
        expect(ACTION_CONSTANTS).toHaveProperty('FORM_SUBMISSION_DELETE');
        expect(ACTION_CONSTANTS).toHaveProperty('FORM_SUBMISSION_ERROR');
        expect(ACTION_CONSTANTS).toHaveProperty('IS_FORM_SUBMISSION_LOADING');
        expect(ACTION_CONSTANTS).toHaveProperty('IS_FORM_WORKFLOW_SAVED');
        expect(ACTION_CONSTANTS).toHaveProperty('PUBLIC_FORM_SUBMIT');
        expect(ACTION_CONSTANTS).toHaveProperty('PUBLIC_FORM_STATUS');
        expect(ACTION_CONSTANTS).toHaveProperty('IS_FORM_LOADING');
        expect(ACTION_CONSTANTS).toHaveProperty('APPLICATION_COUNT');
        expect(ACTION_CONSTANTS).toHaveProperty('IS_APPLICATION_COUNT_LOADING');
        expect(ACTION_CONSTANTS).toHaveProperty('APPLICATION_COUNT_RESPONSE');
        expect(ACTION_CONSTANTS).toHaveProperty('UNPUBLISH_API_ERROR');
        expect(ACTION_CONSTANTS).toHaveProperty('IS_PUBLIC_STATUS_LOADING');
        expect(ACTION_CONSTANTS).toHaveProperty('IS_FORM_SEARCH_LOADING');
    });

    test('should contain the correct properties of custom submission', () => {
        expect(ACTION_CONSTANTS).toHaveProperty('CUSTOM_SUBMISSION');
    });

    test('should contain the correct properties of React Formio Action Constants', () => {
        expect(ACTION_CONSTANTS).toHaveProperty("FORM_CLEAR_ERROR");
        expect(ACTION_CONSTANTS).toHaveProperty("FORM_REQUEST");
        expect(ACTION_CONSTANTS).toHaveProperty("FORM_SUCCESS");
        expect(ACTION_CONSTANTS).toHaveProperty("FORM_FAILURE");
        expect(ACTION_CONSTANTS).toHaveProperty("FORM_SAVE");
        expect(ACTION_CONSTANTS).toHaveProperty("FORM_RESET");
        expect(ACTION_CONSTANTS).toHaveProperty("SUBMISSION_CLEAR_ERROR");
    });

    test('should contain the correct properties of dashboard', () => {
        expect(ACTION_CONSTANTS).toHaveProperty("LIST_DASHBOARDS");
        expect(ACTION_CONSTANTS).toHaveProperty("DASHBOARD_DETAIL");
        expect(ACTION_CONSTANTS).toHaveProperty("IS_DASHBOARD_LOADING");
        expect(ACTION_CONSTANTS).toHaveProperty("IS_INSIGHT_DETAIL_LOADING");
        expect(ACTION_CONSTANTS).toHaveProperty("INSIGHT_ERROR");
        expect(ACTION_CONSTANTS).toHaveProperty("CLEANUP_INSIGHTS");
    });

    test('should contain the correct properties of metrics', () => {
        expect(ACTION_CONSTANTS).toHaveProperty("METRICS_SUBMISSION_DATE");
        expect(ACTION_CONSTANTS).toHaveProperty("METRICS_LIST");
        expect(ACTION_CONSTANTS).toHaveProperty("METRICS_SUBMISSIONS");
        expect(ACTION_CONSTANTS).toHaveProperty("IS_METRICS_LOADING");
        expect(ACTION_CONSTANTS).toHaveProperty("METRICS_SUBMISSIONS_STATUS");
        expect(ACTION_CONSTANTS).toHaveProperty("IS_METRICS_STATUS_LOADING");
        expect(ACTION_CONSTANTS).toHaveProperty("SELECTED_METRICS_ID");
        expect(ACTION_CONSTANTS).toHaveProperty("METRICS_LOAD_ERROR");
        expect(ACTION_CONSTANTS).toHaveProperty("METRICS_STATUS_LOAD_ERROR");
        expect(ACTION_CONSTANTS).toHaveProperty("IS_PROCESS_STATUS_LOADING");
        expect(ACTION_CONSTANTS).toHaveProperty("PROCESS_STATUS_LIST");
        expect(ACTION_CONSTANTS).toHaveProperty("IS_PROCESS_STATUS_LOAD_ERROR");
        expect(ACTION_CONSTANTS).toHaveProperty("IS_PROCESS_ACTIVITY_LOAD_ERROR");
        expect(ACTION_CONSTANTS).toHaveProperty("METRICS_SUBMISSIONS_SEARCH");
        expect(ACTION_CONSTANTS).toHaveProperty("METRICS_SUBMISSIONS_SORT_CHANGE");
        expect(ACTION_CONSTANTS).toHaveProperty("METRICS_SUBMISSIONS_LIST_LIMIT_CHANGE");
        expect(ACTION_CONSTANTS).toHaveProperty("METRICS_SUBMISSIONS_LIST_PAGE_CHANGE");
        expect(ACTION_CONSTANTS).toHaveProperty("METRICS_DATE_RANGE_LOADING");
        expect(ACTION_CONSTANTS).toHaveProperty("METRICS_SUBMISSION_STATUS_COUNT_LOADER");
        expect(ACTION_CONSTANTS).toHaveProperty("METRICS_SUBMISSIONS_COUNT");
    });

    test('should contain the correct properties of application history', () => {
        expect(ACTION_CONSTANTS).toHaveProperty("LIST_APPLICATION_HISTORY");
        expect(ACTION_CONSTANTS).toHaveProperty("APPLICATION_HISTORY_DETAIL");
        expect(ACTION_CONSTANTS).toHaveProperty("PROCESS_LIST");
        expect(ACTION_CONSTANTS).toHaveProperty("DMN_PROCESS_LIST");
        expect(ACTION_CONSTANTS).toHaveProperty("IS_HISTORY_LOADING");
        expect(ACTION_CONSTANTS).toHaveProperty("IS_FORM_PROCESS_STATUS_LOAD_ERROR");
        expect(ACTION_CONSTANTS).toHaveProperty("FORM_PROCESS_LIST");
        expect(ACTION_CONSTANTS).toHaveProperty("WORKFLOW_ASSOCIATION_CHANGED");
    });

    test('should contain the correct properties of application', () => {
        expect(ACTION_CONSTANTS).toHaveProperty("LIST_APPLICATIONS");
        expect(ACTION_CONSTANTS).toHaveProperty("LIST_APPLICATIONS_OF_FORM");
        expect(ACTION_CONSTANTS).toHaveProperty("APPLICATION_DETAIL");
        expect(ACTION_CONSTANTS).toHaveProperty("APPLICATION_DETAIL_STATUS_CODE");
        expect(ACTION_CONSTANTS).toHaveProperty("IS_APPLICATION_LIST_LOADING");
        expect(ACTION_CONSTANTS).toHaveProperty("IS_APPLICATION_DETAIL_LOADING");
        expect(ACTION_CONSTANTS).toHaveProperty("IS_APPLICATION_UPDATING");
        expect(ACTION_CONSTANTS).toHaveProperty("APPLICATION_PROCESS");
        expect(ACTION_CONSTANTS).toHaveProperty("SET_APPLICATION_LIST_COUNT");
        expect(ACTION_CONSTANTS).toHaveProperty("PROCESS_ACTIVITIES");
        expect(ACTION_CONSTANTS).toHaveProperty("PROCESS_DIAGRAM_XML");
        expect(ACTION_CONSTANTS).toHaveProperty("IS_PROCESS_DIAGRAM_LOADING");
        expect(ACTION_CONSTANTS).toHaveProperty("APPLICATION_LIST_ACTIVE_PAGE");
        expect(ACTION_CONSTANTS).toHaveProperty("APPLICATION_STATUS_LIST");
        expect(ACTION_CONSTANTS).toHaveProperty("APPLICATIONS_ERROR");
        expect(ACTION_CONSTANTS).toHaveProperty("FORM_PREVIOUS_DATA");
    });

    test('should contain the correct properties of menu', () => {
        expect(ACTION_CONSTANTS).toHaveProperty("TOGGLE_MENU");
    });

    test('should contain the correct properties of BPM tasks', () => {
        expect(ACTION_CONSTANTS).toHaveProperty("BPM_LIST_TASKS");
        expect(ACTION_CONSTANTS).toHaveProperty("BPM_PROCESS_LIST");
        expect(ACTION_CONSTANTS).toHaveProperty("BPM_USER_LIST");
        expect(ACTION_CONSTANTS).toHaveProperty("BPM_TASKS_COUNT");
        expect(ACTION_CONSTANTS).toHaveProperty("BPM_TASK_DETAIL");
        expect(ACTION_CONSTANTS).toHaveProperty("IS_BPM_TASK_UPDATING");
        expect(ACTION_CONSTANTS).toHaveProperty("IS_BPM_TASK_LOADING");
        expect(ACTION_CONSTANTS).toHaveProperty("IS_BPM_TASK_DETAIL_LOADING");
        expect(ACTION_CONSTANTS).toHaveProperty("IS_BPM_TASK_DETAIL_UPDATING");
        expect(ACTION_CONSTANTS).toHaveProperty("BPM_FILTER_LIST");
        expect(ACTION_CONSTANTS).toHaveProperty("IS_BPM_FILTERS_LOADING");
        expect(ACTION_CONSTANTS).toHaveProperty("BPM_SELECTED_FILTER");
        expect(ACTION_CONSTANTS).toHaveProperty("SELECTED_TASK_ID");
        expect(ACTION_CONSTANTS).toHaveProperty("SET_TASK_GROUP");
        expect(ACTION_CONSTANTS).toHaveProperty("IS_TASK_GROUP_LOADING");
        expect(ACTION_CONSTANTS).toHaveProperty("UPDATE_FILTER_LIST_SORT_PARAMS");
        expect(ACTION_CONSTANTS).toHaveProperty("UPDATE_FILTER_LIST_SEARCH_PARAMS");
        expect(ACTION_CONSTANTS).toHaveProperty("UPDATE_LIST_PARAMS");
        expect(ACTION_CONSTANTS).toHaveProperty("UPDATE_SEARCH_QUERY_TYPE");
        expect(ACTION_CONSTANTS).toHaveProperty("UPDATE_VARIABLE_NAME_IGNORE_CASE");
        expect(ACTION_CONSTANTS).toHaveProperty("UPDATE_VARIABLE_VALUE_IGNORE_CASE");
        expect(ACTION_CONSTANTS).toHaveProperty("RELOAD_TASK_FORM_SUBMISSION");
        expect(ACTION_CONSTANTS).toHaveProperty("BPM_TASK_LIST_ACTIVE_PAGE");
        expect(ACTION_CONSTANTS).toHaveProperty("DEPLOYMENT_LIST");
    });

    test('should have the correct BPM FORM actions', () => {
        expect(ACTION_CONSTANTS).toHaveProperty('BPM_FORM_LIST');
        expect(ACTION_CONSTANTS).toHaveProperty('IS_BPM_FORM_LIST_LOADING');
        expect(ACTION_CONSTANTS).toHaveProperty('BPM_FORM_LIST_PAGE_CHANGE');
        expect(ACTION_CONSTANTS).toHaveProperty('BPM_FORM_LIST_LIMIT_CHANGE');
        expect(ACTION_CONSTANTS).toHaveProperty('BPM_FORM_LIST_SORT_CHANGE');
        expect(ACTION_CONSTANTS).toHaveProperty('BPM_MAINTAIN_PAGINATION');
        expect(ACTION_CONSTANTS).toHaveProperty('BPM_FORM_SEARCH');
        expect(ACTION_CONSTANTS).toHaveProperty('BPM_FORM_TYPE');
        expect(ACTION_CONSTANTS).toHaveProperty('BPM_FORM_LOADING');
    });

    test('should have the correct CHECKLIST FORM actions', () => {
        expect(ACTION_CONSTANTS).toHaveProperty('FORM_CHECK_LIST_UPDATE');
        expect(ACTION_CONSTANTS).toHaveProperty('FORM_UPLOAD_LIST');
        expect(ACTION_CONSTANTS).toHaveProperty('FORM_UPLOAD_COUNTER');
        expect(ACTION_CONSTANTS).toHaveProperty('FORM_UPLOAD_FAILURE_COUNTER');
        expect(ACTION_CONSTANTS).toHaveProperty('CHANGE_SIZE_PER_PAGE');
    });

    test('should have the correct DASHBOARD actions', () => {
        expect(ACTION_CONSTANTS).toHaveProperty('DASHBOARDS_LIST');
        expect(ACTION_CONSTANTS).toHaveProperty('DASHBOARDS_LIST_ERROR');
        expect(ACTION_CONSTANTS).toHaveProperty('DASHBOARDS_LIST_GROUPS');
        expect(ACTION_CONSTANTS).toHaveProperty('DASHBOARDS_UPDATE_ERROR');
        expect(ACTION_CONSTANTS).toHaveProperty('SET_AUTHORIZATIONS');
        expect(ACTION_CONSTANTS).toHaveProperty('UPDATE_AUTHORIZATIONS');
    });

    test('should have the correct PROCESS actions', () => {
        expect(ACTION_CONSTANTS).toHaveProperty('RESET_PROCESS');
        expect(ACTION_CONSTANTS).toHaveProperty('FORM_STATUS_LOADING');
    });

    test('should have the correct TENANT actions', () => {
        expect(ACTION_CONSTANTS).toHaveProperty('RESET_TENANT');
        expect(ACTION_CONSTANTS).toHaveProperty('SET_TENANT_DETAILS');
        expect(ACTION_CONSTANTS).toHaveProperty('SET_TENANT_ID');
        expect(ACTION_CONSTANTS).toHaveProperty('IS_TENANT_DETAIL_LOADING');
        expect(ACTION_CONSTANTS).toHaveProperty('SET_TENANT_DATA');
    });

    test('should have the correct DRAFT actions', () => {
        expect(ACTION_CONSTANTS).toHaveProperty('SAVE_DRAFT_DATA');
        expect(ACTION_CONSTANTS).toHaveProperty('DRAFT_SUBMISSION_ERROR');
        expect(ACTION_CONSTANTS).toHaveProperty('DRAFT_LIST');
        expect(ACTION_CONSTANTS).toHaveProperty('DRAFT_DETAIL');
        expect(ACTION_CONSTANTS).toHaveProperty('DRAFT_COUNT');
        expect(ACTION_CONSTANTS).toHaveProperty('DRAFT_LIST_LOADER');
        expect(ACTION_CONSTANTS).toHaveProperty('SET_DRAFT_LIST_ACTIVE_PAGE');
        expect(ACTION_CONSTANTS).toHaveProperty('SET_DRAFT_COUNT_PER_PAGE');
        expect(ACTION_CONSTANTS).toHaveProperty('DRAFT_DETAIL_STATUS_CODE');
        expect(ACTION_CONSTANTS).toHaveProperty('DRAFT_LAST_UPDATED');
        expect(ACTION_CONSTANTS).toHaveProperty('DRAFT_DELETE');
    });

});
