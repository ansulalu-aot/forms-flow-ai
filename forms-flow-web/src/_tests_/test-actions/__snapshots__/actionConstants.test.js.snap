// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ACTION_CONSTANTS should match the snapshot 1`] = `
Object {
  "ACCESS_ADDING": "ACCESS_ADDING",
  "APPLICATIONS_ERROR": "APPLICATIONS_ERROR",
  "APPLICATION_COUNT": "APPLICATION_COUNT",
  "APPLICATION_COUNT_RESPONSE": "APPLICATION_COUNT_RESPONSE",
  "APPLICATION_DETAIL": "APPLICATION_DETAIL",
  "APPLICATION_DETAIL_STATUS_CODE": "APPLICATION_DETAIL_STATUS_CODE",
  "APPLICATION_HISTORY_DETAIL": "APPLICATION_HISTORY_DETAIL",
  "APPLICATION_LIST_ACTIVE_PAGE": "APPLICATION_LIST_ACTIVE_PAGE",
  "APPLICATION_PROCESS": "APPLICATION_PROCESS",
  "APPLICATION_STATUS_LIST": "APPLICATION_STATUS_LIST",
  "BPM_FILTER_LIST": "BPM_FILTER_LIST",
  "BPM_FORM_LIST": "BPM_FORM_LIST",
  "BPM_FORM_LIST_LIMIT_CHANGE": "BPM_FORM_LIST_LIMIT_CHANGE",
  "BPM_FORM_LIST_PAGE_CHANGE": "BPM_FORM_LIST_PAGE_CHANGE",
  "BPM_FORM_LIST_SORT_CHANGE": "BPM_FORM_LIST_SORT_CHANGE",
  "BPM_FORM_LOADING": "BPM_FORM_LOADING",
  "BPM_FORM_SEARCH": "BPM_FORM_SEARCH",
  "BPM_FORM_TYPE": "BPM_FORM_TYPE",
  "BPM_LIST_TASKS": "BPM_LIST_TASKS",
  "BPM_MAINTAIN_PAGINATION": "BPM_MAINTAIN_PAGINATION",
  "BPM_PROCESS_LIST": "BPM_PROCESS_LIST",
  "BPM_SELECTED_FILTER": "BPM_SELECTED_FILTER",
  "BPM_TASKS_COUNT": "BPM_TASKS_COUNT",
  "BPM_TASK_DETAIL": "BPM_TASK_DETAIL",
  "BPM_TASK_LIST_ACTIVE_PAGE": "BPM_TASK_LIST_ACTIVE_PAGE",
  "BPM_USER_LIST": "BPM_USER_LIST",
  "CHANGE_SIZE_PER_PAGE": "CHANGE_SIZE_PER_PAGE",
  "CLEANUP_INSIGHTS": "CLEANUP_INSIGHTS",
  "CUSTOM_SUBMISSION": "CUSTOM_SUBMISSION",
  "DASHBOARDS_LIST": "DASHBOARDS_LIST",
  "DASHBOARDS_LIST_ERROR": "DASHBOARDS_LIST_ERROR",
  "DASHBOARDS_LIST_GROUPS": "DASHBOARDS_LIST_GROUPS",
  "DASHBOARDS_UPDATE_ERROR": "DASHBOARDS_UPDATE_ERROR",
  "DASHBOARD_DETAIL": "DASHBOARD_DETAIL",
  "DEPLOYMENT_LIST": "DEPLOYMENT_LIST",
  "DMN_PROCESS_LIST": "DMN_PROCESS_LIST",
  "DRAFT_COUNT": "DRAFT_COUNT",
  "DRAFT_DELETE": "DRAFT_DELETE",
  "DRAFT_DETAIL": "DRAFT_DETAIL",
  "DRAFT_DETAIL_STATUS_CODE": "DRAFT_DETAIL_STATUS_CODE",
  "DRAFT_LAST_UPDATED": "DRAFT_LAST_UPDATED",
  "DRAFT_LIST": "DRAFT_LIST",
  "DRAFT_LIST_LOADER": "DRAFT_LIST_LOADER",
  "DRAFT_SUBMISSION_ERROR": "DRAFT_SUBMISSION_ERROR",
  "ERROR": "ERROR",
  "FORM_CHECK_LIST_UPDATE": "FORM_CHECK_LIST_UPDATE",
  "FORM_CLEAR_ERROR": "FORM_CLEAR_ERROR",
  "FORM_DELETE": "FORM_DELETE",
  "FORM_FAILURE": "FORM_FAILURE",
  "FORM_HISTORY": "FORM_HISTORY",
  "FORM_PREVIOUS_DATA": "FORM_PREVIOUS_DATA",
  "FORM_PROCESS_LIST": "FORM_PROCESS_LIST",
  "FORM_REQUEST": "FORM_REQUEST",
  "FORM_RESET": "FORM_RESET",
  "FORM_SAVE": "FORM_SAVE",
  "FORM_STATUS_LOADING": "FORM_STATUS_LOADING",
  "FORM_SUBMISSION_DELETE": "FORM_SUBMISSION_DELETE",
  "FORM_SUBMISSION_ERROR": "FORM_SUBMISSION_ERROR",
  "FORM_SUCCESS": "FORM_SUCCESS",
  "FORM_UPLOAD_COUNTER": "FORM_UPLOAD_COUNTER",
  "FORM_UPLOAD_FAILURE_COUNTER": "FORM_UPLOAD_FAILURE_COUNTER",
  "FORM_UPLOAD_LIST": "FORM_UPLOAD_LIST",
  "INSIGHT_ERROR": "INSIGHT_ERROR",
  "IS_APPLICATION_COUNT_LOADING": "IS_APPLICATION_COUNT_LOADING",
  "IS_APPLICATION_DETAIL_LOADING": "IS_APPLICATION_DETAIL_LOADING",
  "IS_APPLICATION_LIST_LOADING": "IS_APPLICATION_LIST_LOADING",
  "IS_APPLICATION_UPDATING": "IS_APPLICATION_UPDATING",
  "IS_BPM_FILTERS_LOADING": "IS_BPM_FILTERS_LOADING",
  "IS_BPM_FORM_LIST_LOADING": "IS_BPM_FORM_LIST_LOADING",
  "IS_BPM_TASK_DETAIL_LOADING": "IS_BPM_TASK_DETAIL_LOADING",
  "IS_BPM_TASK_DETAIL_UPDATING": "IS_BPM_TASK_DETAIL_UPDATING",
  "IS_BPM_TASK_LOADING": "IS_BPM_TASK_LOADING",
  "IS_BPM_TASK_UPDATING": "IS_BPM_TASK_UPDATING",
  "IS_DASHBOARD_LOADING": "IS_DASHBOARD_LOADING",
  "IS_FORM_LOADING": "IS_FORM_LOADING",
  "IS_FORM_PROCESS_STATUS_LOAD_ERROR": "IS_FORM_PROCESS_STATUS_LOAD_ERROR",
  "IS_FORM_SEARCH_LOADING": "IS_FORM_SEARCH_LOADING",
  "IS_FORM_SUBMISSION_LOADING": "IS_FORM_SUBMISSION_LOADING",
  "IS_FORM_WORKFLOW_SAVED": "IS_FORM_WORKFLOW_SAVED",
  "IS_HISTORY_LOADING": "IS_HISTORY_LOADING",
  "IS_INSIGHT_DETAIL_LOADING": "IS_INSIGHT_DETAIL_LOADING",
  "IS_LOADING": "IS_LOADING",
  "IS_METRICS_LOADING": "IS_METRICS_LOADING",
  "IS_METRICS_STATUS_LOADING": "IS_METRICS_STATUS_LOADING",
  "IS_PROCESS_ACTIVITY_LOAD_ERROR": "IS_PROCESS_ACTIVITY_LOAD_ERROR",
  "IS_PROCESS_DIAGRAM_LOADING": "IS_PROCESS_DIAGRAM_LOADING",
  "IS_PROCESS_STATUS_LOADING": "IS_PROCESS_STATUS_LOADING",
  "IS_PROCESS_STATUS_LOAD_ERROR": "IS_PROCESS_STATUS_LOAD_ERROR",
  "IS_PUBLIC_STATUS_LOADING": "IS_PUBLIC_STATUS_LOADING",
  "IS_TASK_GROUP_LOADING": "IS_TASK_GROUP_LOADING",
  "IS_TENANT_DETAIL_LOADING": "IS_TENANT_DETAIL_LOADING",
  "LIST_APPLICATIONS": "LIST_APPLICATIONS",
  "LIST_APPLICATIONS_OF_FORM": "LIST_APPLICATIONS_OF_FORM",
  "LIST_APPLICATION_HISTORY": "LIST_APPLICATION_HISTORY",
  "LIST_DASHBOARDS": "LIST_DASHBOARDS",
  "METRICS_DATE_RANGE_LOADING": "METRICS_DATE_RANGE_LOADING",
  "METRICS_LIST": "METRICS_LIST",
  "METRICS_LOAD_ERROR": "METRICS_LOAD_ERROR",
  "METRICS_STATUS_LOAD_ERROR": "METRICS_STATUS_LOAD_ERROR",
  "METRICS_SUBMISSIONS": "METRICS_SUBMISSIONS",
  "METRICS_SUBMISSIONS_COUNT": "METRICS_SUBMISSIONS_COUNT",
  "METRICS_SUBMISSIONS_LIST_LIMIT_CHANGE": "METRICS_SUBMISSIONS_LIST_LIMIT_CHANGE",
  "METRICS_SUBMISSIONS_LIST_PAGE_CHANGE": "METRICS_SUBMISSIONS_LIST_PAGE_CHANGE",
  "METRICS_SUBMISSIONS_SEARCH": "METRICS_SUBMISSIONS_SEARCH",
  "METRICS_SUBMISSIONS_SORT_CHANGE": "METRICS_SUBMISSIONS_SORT_CHANGE",
  "METRICS_SUBMISSIONS_STATUS": "METRICS_SUBMISSIONS_STATUS",
  "METRICS_SUBMISSION_DATE": "METRICS_SUBMISSION_DATE",
  "METRICS_SUBMISSION_STATUS_COUNT_LOADER": "METRICS_SUBMISSION_STATUS_COUNT_LOADER",
  "PROCESS_ACTIVITIES": "PROCESS_ACTIVITIES",
  "PROCESS_DIAGRAM_XML": "PROCESS_DIAGRAM_XML",
  "PROCESS_LIST": "PROCESS_LIST",
  "PROCESS_STATUS_LIST": "PROCESS_STATUS_LIST",
  "PUBLIC_FORM_STATUS": "PUBLIC_FORM_STATUS",
  "PUBLIC_FORM_SUBMIT": "IS_FORM_SUBMITTED",
  "RELOAD_TASK_FORM_SUBMISSION": "RELOAD_TASK_FORM_SUBMISSION",
  "RESET_PROCESS": "RESET_PROCESS",
  "RESET_TENANT": "RESET_TENANT",
  "RESTORE_FORM_DATA": "RESTORE_FORM_DATA",
  "RESTORE_FORM_ID": "RESTORE_FORM_ID",
  "ROLE_IDS": "ROLE_IDS",
  "SAVE_DRAFT_DATA": "SAVE_DRAFT_DATA",
  "SELECTED_METRICS_ID": "SELECTED_METRICS_ID",
  "SELECTED_TASK_ID": "SELECTED_TASK_ID",
  "SET_APPLICATION_LIST_COUNT": "SET_APPLICATION_LIST_COUNT",
  "SET_AUTHORIZATIONS": "SET_AUTHORIZATIONS",
  "SET_CURRENT_PAGE": "SET_CURRENT_PAGE",
  "SET_DRAFT_COUNT_PER_PAGE": "SET_DRAFT_COUNT_PER_PAGE",
  "SET_DRAFT_LIST_ACTIVE_PAGE": "SET_DRAFT_LIST_ACTIVE_PAGE",
  "SET_LANGUAGE": "SET_LANGUAGE",
  "SET_SELECT_LANGUAGES": "SET_SELECT_LANGUAGES",
  "SET_TASK_GROUP": "SET_TASK_GROUP",
  "SET_TENANT_DATA": "SET_TENANT_DATA",
  "SET_TENANT_DETAILS": "SET_TENANT_DETAILS",
  "SET_TENANT_ID": "SET_TENANT_ID",
  "SET_USER_AUTHENTICATION": "SET_USER_AUTHENTICATION",
  "SET_USER_DETAILS": "SET_USER_DETAILS",
  "SET_USER_ROLES": "SET_USER_ROLES",
  "SET_USER_TOKEN": "SET_USER_TOKEN",
  "SUBMISSION_CLEAR_ERROR": "SUBMISSION_CLEAR_ERROR",
  "TOGGLE_MENU": "TOGGLE_MENU",
  "UNPUBLISH_API_ERROR": "UNPUBLISH_API_ERROR",
  "UPDATE_AUTHORIZATIONS": "UPDATE_AUTHORIZATIONS",
  "UPDATE_FILTER_LIST_SEARCH_PARAMS": "UPDATE_FILTER_LIST_SEARCH_PARAMS",
  "UPDATE_FILTER_LIST_SORT_PARAMS": "UPDATE_FILTER_LIST_SORT_PARAMS",
  "UPDATE_LIST_PARAMS": "UPDATE_LIST_PARAMS",
  "UPDATE_SEARCH_QUERY_TYPE": "UPDATE_SEARCH_QUERY_TYPE",
  "UPDATE_VARIABLE_NAME_IGNORE_CASE": "UPDATE_VARIABLE_NAME_IGNORE_CASE",
  "UPDATE_VARIABLE_VALUE_IGNORE_CASE": "UPDATE_VARIABLE_VALUE_IGNORE_CASE",
  "WORKFLOW_ASSOCIATION_CHANGED": "WORKFLOW_ASSOCIATION_CHANGED",
}
`;
